serenata_toolbox/datasets/remote.py

```diff
@@ -0,0 +1,34 @@
+import configparser
+import os
+
+import boto3
+
+from serenata_toolbox.datasets.contextmanager import status_message
+
+
+class RemoteDatasets:
+
+    def __init__(self):
```

Comment:
maybe it is necessary to add a test to check if the `config.ini` file exists would be nice to give people clues why it is not working properly =)